### ISSUE TITLE
[bug] ember/no-restricted-resolver-tests false positive

### ISSUE THREAD

**Issue by mike-north** [ember-cli/eslint-plugin-ember#343]
@scalvert's #278 introduced a rule that fails on any function `setupTest` that's passed a single argument

https://github.com/ember-cli/eslint-plugin-ember/blob/88b340e060be65972a4a2c94ad2fddda7fb46d66/lib/rules/no-restricted-resolver-tests.js#L87-L120

`ember-fastboot-addon-tests`, a `devDependency` for some top 10% [popular](https://github.com/kimroen/ember-cli-document-title) [addons](https://github.com/simplabs/ember-simple-auth) has had a `setupTest` function for two years.

https://github.com/kaliber5/ember-fastboot-addon-tests/blob/c6f3970c6cb29b241de8c43bfe7eed767578cfe3/index.js#L15

As a result, any projects that use `ember-fastboot-addon-tests` may error upon upgrading `eslint-plugin-ember`.

### Workaround

Every addon that encounters this problem can 
```
/* eslint-disable ember/no-restricted-resolver-tests */
```

### Ideal fix

the `no-restricted-resolver-tests` rule should be narrowed so that it only alerts on the correct `setupTest` functions, as imported from specific libraries the rule is intended to target

**Comment by scalvert** [comment]
This seems entirely reasonable. We can inspect the imports and determine whether the path is correct.
